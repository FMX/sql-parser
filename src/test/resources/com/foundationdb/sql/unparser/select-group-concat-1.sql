SELECT a, GROUP_CONCAT(b) FROM t GROUP BY a