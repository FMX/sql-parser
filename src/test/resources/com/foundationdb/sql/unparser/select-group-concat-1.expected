SELECT a, (GROUP_CONCAT(b SEPARATOR ',')) FROM t GROUP BY a